civis/io/_tables.py

```diff
@@ -981,6 +1001,9 @@ def civis_file_to_table(file_id, database, table, client=None,
         file_id = [file_id]
     if schema is None:
         raise ValueError("Provide a schema as part of the `table` input.")
+    if table_columns is not None and not isinstance(table_columns, list):
+        raise ValueError("Provide table_columns as a list of dictionaries"
+                         " with keys of `name` and `sqlType`.")
```

Comment:
We shouldn't check if `table_columns` is a `list`. If we're going to implement type checking in the client then we should do that everywhere.

If we wanted to check if `table_columns` has keys of `name` and `sqlType` then I would be okay with that but I think its unnecessary because the API will tell us thats the problem. Now someone will have to maintain this code in the client and update it when the API is updated versus just using the API.

For reference, the `fut` should return the below error when a `table_columns` is passed without the required keys (e.g., `sqlTypes`).

![image](https://user-images.githubusercontent.com/4155284/78390104-96507700-75a9-11ea-8dac-cb75285c3d18.png)

Reply:
@mheilman @zacharydfriedlander fyi

Reply:
oh, nice.  i'm glad the error is pretty informative